Drivers/seds/UartHwasComponent.xml

```diff
@@ -0,0 +1,1251 @@
+                    <VariableRef variableRef="uartVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="config">
+                    <VariableRef variableRef="configVar" />
+                  </ArgumentValue>
+                </Call>
+                <SendCommandPrimitive shortDescription="Return set uart container" interface="UartHwas" command="InitUartCmd">
+                  <ArgumentValue name="uart">
+                    <VariableRef variableRef="uartVar" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+              </Body>
+            </Activity>
+            <Activity name="ReadByteAsync">
+              <Argument name="uart" type="com/n7space/UartHwas/UartHwas" />
+              <Body>
+                <Assignment outputVariableRef="destinationAddressVar">
+                  <VariableRef variableRef="uart.mUartIerAddress" />
+                </Assignment>
+                <SendCommandPrimitive shortDescription="Enable Rx ready interrupt" interface="RawMemoryAccess" command="WriteWord">
+                  <ArgumentValue name="address">
+                    <VariableRef variableRef="destinationAddressVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="mask">
+                    <Value value="1" />
+                  </ArgumentValue>
+                  <ArgumentValue name="inputValue">
+                    <Value value="1" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+                <Assignment outputVariableRef="irqNumberVar">
+                  <VariableRef variableRef="uartVar.mIrqNumber" />
+                </Assignment>
+                <SendCommandPrimitive shortDescription="Subscribe to Uart interrupt" interface="InterruptSubscriptionManagement" command="SubscribeToInterrupt">
+                  <ArgumentValue name="interrupt">
+                    <VariableRef variableRef="irqNumberVar" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+                <SendCommandPrimitive shortDescription="Enable Uart interrupt" interface="InterruptManagement" command="EnableInterrupt">
+                  <ArgumentValue name="interrupt">
+                    <VariableRef variableRef="irqNumberVar" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+              </Body>
+            </Activity>
+            <Activity name="SendByteAsync">
+              <Argument name="uart" type="com/n7space/UartHwas/UartHwas"></Argument>
+              <Argument name="byte" type="com/n7space/Hwas/Byte"></Argument>
+              <Body>
+                <Assignment outputVariableRef="irqNumberVar">
+                  <VariableRef variableRef="uartVar.mIrqNumber" />
+                </Assignment>
+                <SendCommandPrimitive shortDescription="Subscribe to Uart interrupt" interface="InterruptSubscriptionManagement" command="SubscribeToInterrupt">

```

Comment:
Are the changes from the previous MR propagated to this one?

Reply:
The name of this file has changed, and it was split into two files. I guess that's why changes were not propagated. I will try to sync this branch with the master.

Reply:
done